File: CRM/src/app/shared/eHospital_DongNai_A/TiepNhan_ThuocUngThu.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { TiepNhan_ThuocUngThu } from './TiepNhan_ThuocUngThu.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class TiepNhan_ThuocUngThuService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'TiepNhan_ThuocUngThu_Id', 'TiepNhan_Id', 'DuocPhepSuDung', 'GiayChungNhan', 'NgayTao', 'NguoiTao_Id', 'NgayCapNhat', 'NguoiCapNhat_Id'];

    List: TiepNhan_ThuocUngThu[] | undefined;
    ListFilter: TiepNhan_ThuocUngThu[] | undefined;
    FormData!: TiepNhan_ThuocUngThu;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "TiepNhan_ThuocUngThu";
    }
}

File: Service_eHospital_DongNai_A/Implement/TiepNhan_ThuocUngThuService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class TiepNhan_ThuocUngThuService : BaseService<TiepNhan_ThuocUngThu, ITiepNhan_ThuocUngThuRepository>
    , ITiepNhan_ThuocUngThuService
    {
    private readonly ITiepNhan_ThuocUngThuRepository _TiepNhan_ThuocUngThuRepository;
    public TiepNhan_ThuocUngThuService(ITiepNhan_ThuocUngThuRepository TiepNhan_ThuocUngThuRepository) : base(TiepNhan_ThuocUngThuRepository)
    {
    _TiepNhan_ThuocUngThuRepository = TiepNhan_ThuocUngThuRepository;
    }
    }
    }

